### PR TITLE
Allow to produce stable version of TA and MSTest independently

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: eng/Build.props [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+  <ItemGroup Condition=" '$(StableVersionComponent)' == 'mstest' AND '$(OS)' != 'Windows_NT' ">
+    <ProjectToBuild Include="$(RepoRoot)src/Adapter/**/*.csproj" />
+    <ProjectToBuild Include="$(RepoRoot)src/Analyzers/**/*.csproj" />
+    <ProjectToBuild Include="$(RepoRoot)src/Package/**/*.csproj" />
+    <ProjectToBuild Include="$(RepoRoot)src/TestFramework/**/*.csproj" />
+    <ProjectToBuild Include="$(RepoRoot)test/UnitTests/MSTest.Analyzers.UnitTests/MSTest.Analyzers.UnitTests.csproj" />
+    <ProjectToBuild Include="$(RepoRoot)test/IntegrationTests/MSTest.Acceptance.IntegrationTests/MSTest.Acceptance.IntegrationTests.csproj" />
+  </ItemGroup>
+
+  <!-- For stable builds of TestAnywhere -->
```
Comment (fhnaseer) [inline eng/Build.props]: Maybe use `TestingPlatform` instead of `TestAnywhere`

File: azure-pipelines-official.yml [hunk 2]
```diff
@@ -29,10 +29,14 @@ pr:
...
-  type: boolean
-  default: False
+- name: stableVersionComponent
+  displayName: "Produce stable/RTM version for specified component (default is none)"
+  type: string
+  default: none
+  values:
+  - none
+  - mstest
+  - testanywhere
```
Comment (Evangelink) [inline azure-pipelines-official.yml]:
```suggestion
  - testing-platform
```

File: eng/Build.props (hunk 1 above)
Comment (Evangelink) [reply eng/Build.props]:
```suggestion
  <!-- For stable builds of Microsoft.Testing.Platform and extensions -->
```

File: eng/Build.props [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
+  <ItemGroup Condition=" '$(StableVersionComponent)' == 'testanywhere' ">
```
Comment (Evangelink) [inline eng/Build.props]:
```suggestion
  <ItemGroup Condition=" '$(StableVersionComponent)' == 'testing-platform' ">
```

**Pull request merged**